src/log/Log.cc

```diff
@@ -178,6 +179,26 @@ void Log::stop_graylog()
   m_graylog.reset();
 }
 
+void Log::set_journald_level(int log, int crash)
+{
+  std::scoped_lock lock(m_flush_mutex);
+  m_journald_log = log;
+  m_journald_crash = crash;
+}
+
+void Log::start_journald()
+{
+  std::scoped_lock lock(m_flush_mutex);
+  if (! m_journald.get())
+    m_journald = std::make_shared<Journald>(m_subs);
+}
+
+void Log::stop_journald()
```

Comment:
this method name is misleading, journald is not stopped here. could you rename it to `stop_journald_logger()`?